test/kubemark/start-kubemark.sh

```diff
@@ -275,32 +345,63 @@ if [[ "${SCALEOUT_CLUSTER:-false}" == "true" ]]; then
     export TENANT_PARTITION_SEQUENCE=${tp_num}
     create-kubemark-master
 
-    if [[ "${KUBERNETES_SCALEOUT_PROXY}" == "true" ]]; then
-      export KUBERNETES_SCALEOUT_PROXY=false
-      PROXY_RESERVED_IP=$(grep server "${PROXY_KUBECONFIG}" | awk -F "/" '{print $3}' | cut -d: -f1)
-      export PROXY_RESERVED_IP
+    export PROXY_RESERVED_IP=$(cat ${KUBE_TEMP}/proxy-reserved-ip.txt)
```

Comment:
We should not use tmp files this way. I have cleaned this up, we should get it from kubeconfig.

Reply:
the tmp file as a media to get subprocess returns is a typical convenient way in bash.